draft-ietf-quic-ack-frequency.md

```diff
@@ -335,17 +330,32 @@ send an acknowledgement immediately on receiving a reordered ack-eliciting
 packet. This extension modifies this behavior.
 
 If the endpoint has not yet received an ACK_FREQUENCY frame, or if the most
-recent frame received from the peer has an `Ignore Order` value of `false`
-(0x00), the endpoint MUST immediately acknowledge any subsequent packets that
+recent frame received from the peer has a `Reordering Threshold` value that is
+not 0x00, the endpoint MUST immediately acknowledge any subsequent packets that
 are received out of order.
 
-If the most recent ACK_FREQUENCY frame received from the peer has an `Ignore
-Order` value of `true` (0x01), the endpoint does not make this exception. That
+If the most recent ACK_FREQUENCY frame received from the peer has a `Reordering
+Threshold` value of 0x00, the endpoint does not make this exception. That
 is, the endpoint MUST NOT send an immediate acknowledgement in response to
 packets received out of order, and instead continues to use the peer's
 `Ack-Eliciting Threshold` and `max_ack_delay` thresholds for sending
 acknowledgements.
 
+### Reordering Threshold
+
+If the most recent value of 'Reordering Threshold' is not 0x00, then in addition
+to the immediate acknowledgement when reordering is detected, an additional
+immediate acknowledgement is sent when the peer can detect missing packets as
+lost.  This can speed up time threshold loss detection and therefore loss
```

Comment:
Unsure what is intended by /time threshold loss detection/ refers to? --is this about the time taken for a sender to detect loss?

Reply:
Suggest: /This can reduce the time to detect loss and therefore improve the performance of loss recovery./